docs/sources/mimir/send/native-histograms/_index.md

```diff
@@ -0,0 +1,268 @@
+---
+description: Learn how to send metric data to Grafana Mimir.
```

Comment:
@knylander-grafana, here is an example of content that we would need to conditionalize for _Learn how to send metric data to Grafana Mimir._ and _Learn how to send metric data to Grafana Cloud Metrics._.

How do you recommend doing that? If a solution is not feasible at this time, we could also leave off the end in favor of _Learn how to send metric data._

Reply:
Similar question for [line 10](https://github.com/grafana/mimir/pull/5956/files#diff-b66a3053778bcea35b3d9b984875cb2cf907a34b597ac5ab8d8fdfd8d9bb2151R10), where we tell the reader where they are sending data to.

Reply:
Maybe we can simplify to say Prometheus remote write endpoint, because that can be Mimir/Cloud Metrics and many other products - not just Grafana made

```suggestion
description: Learn how to send native histograms to a Prometheus remote write endpoint.
```